src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx

```diff
@@ -43,7 +44,15 @@ function WorkspaceExpensifyCardListPage({route, cardsList}: WorkspaceExpensifyCa
 
     const policyID = route.params.policyID;
     const policy = usePolicy(policyID);
+    const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID);
     const [personalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
+    const [cardSettings] = useOnyx(`${ONYXKEYS.COLLECTION.PRIVATE_EXPENSIFY_CARD_SETTINGS}${workspaceAccountID}`);
+
+    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? 0;
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+
+    // TODO: change approvedBy check for more accurate verification status
+    const isBankAccountVerified = bankAccountsList?.[paymentBankAccountID]?.accountData?.approvedBy === 'Automatic';
```

Comment:
How should I check if the account is verified or not?

Reply:
I need to double check this, I think the bank account state wont be open but verifying but I am not 100% sure

Reply:
Ok yeah I think it should be in `VERIFYING` state so you need to check the state of the linked bank account is that

Reply:
@mountiny so can I just check if the bank account is in OPEN state? For other states like PENDING I guess we don't want to display the buttons either

Reply:
@koko57 that is a fair assumption, only the open state should be able to create cards